### PR TITLE
Initial draft

### PR DESCRIPTION
**Pull request by everett-toews**

### CODE REVIEW

File: _tutorials/overview-of-carina.md [hunk 1]
```diff
@@ -1,88 +1,65 @@
...
-* Ensure that your Rackspace Cloud account is enabled for Carinas.
-* Familiarize yourself with basic Docker concepts. Read the [Docker documentation](https://docs.docker.com/
-) and [Docker 101] (docs/tutorials/docker101).
-* Familiarize yourself with the basic container concepts. Read the [Containers 101](docs/tutorials/containers101) document.
-* Familiarize yourself with [scheduling strategies](https://docs.docker.com/swarm/scheduler/strategy/) and with [filters](https://docs.docker.com/swarm/scheduler/filter/).
-* Learn about the resources and resource limitations of a cluster.
-* Install the Docker client on your computer. For installation instructions, see the [Install the Docker client] (link_Install_Docker_Client) section.
-* If you already have the Docker client installed, ensure that you have the right version. Carina, currently supports Docker version 1.8.2.
+Carina is a container runtime environment (currently in Beta) that offers bare metal speeds, security, and portability without sacrificing ease of use. You can get started in minutes using open source software on managed infrastructure to run your containerized applications.
```
Comment (jnoller) [inline _tutorials/overview-of-carina.md]: Should stick to performance, native tooling, ease of use (omit securuity) - see front page for the 3 "sells"

File: _tutorials/overview-of-carina.md [hunk 2]
```diff
@@ -1,88 +1,65 @@
...
 
-#What is Carina?
-**Carina** provides a high-performance service for running and managing Docker containers in a cluster-based infrastructure. Carina gives you a scalable cluster management infrastructure out of the box and allows you to deploy your applications by utilizing one ore more Docker containers across a managed cluster. Carina eliminates the need to install cluster management software and to deploy and operate cluster management hardware. With Carina, you can get started right away and get your containerized application running in a short time.
+Your containers will run in a bare-metal environment avoiding the "hypervisor tax" on performance and run applications as much as 60% faster. That environment is locked down by AppArmor to keep your resources isolated.
```
Comment (jnoller) [inline _tutorials/overview-of-carina.md]: Should you cite @jamiehannaford's work here or include / copy paste the internals / layers - lxc/libvirt apparmor all provide layers of protection

File: _tutorials/overview-of-carina.md [hunk 3]
```diff
@@ -1,88 +1,65 @@
...
 
-The following diagram outlines how Carina works.
+Carina is built on the open source Docker Swarm project. It exposes the native Docker API so you get maximum portability to easily move applications from development to test and production environments, reducing errors, and saving time.
```
Comment (jnoller) [inline _tutorials/overview-of-carina.md]: open source Magnum project exposing the native Docker/Swarm API

File: _tutorials/overview-of-carina.md [hunk 4]
```diff
@@ -1,88 +1,65 @@
...
 
-[Insert the diagram that Nate is working on here]
+You'll also have access to an intuitive user interface and Carina specific developer tooling in addition to the entire ecosystem of tools already compatible with the Docker API. There is also a wealth of documentation, from getting started guides to detailed tutorials and best practices. And if you need help, there's community support directly from other developers.
 
+From creating an account to running a containerized application on a cluster takes minutes. You'll be using open source software like Docker to compose your applications. The infrastructure is managed by Rackspace so you can focus on what's important to you, your business and your applications.
```
Comment (jnoller) [inline _tutorials/overview-of-carina.md]: takes under a minute

File: _tutorials/overview-of-carina.md (hunk 1 above)
Comment (everett-toews) [inline _tutorials/overview-of-carina.md]: Done

File: _tutorials/overview-of-carina.md (hunk 2 above)
Comment (everett-toews) [inline _tutorials/overview-of-carina.md]: Copy/Pasted first sentence from Jamie's article. Link to article in the Resources section.

File: _tutorials/overview-of-carina.md (hunk 3 above)
Comment (everett-toews) [inline _tutorials/overview-of-carina.md]: Leaving off Magnum for the time being.

File: _tutorials/overview-of-carina.md (hunk 4 above)
Comment (everett-toews) [inline _tutorials/overview-of-carina.md]: In my timings, cluster creation takes ~40-45 seconds. Account creation with SMS is going to take at least ~15 seconds. I'm comfortable changing this to "takes under 2 minutes."

**Pull request merged**